Guides/NUI/Components: Added namespace info and deprected note (#1274)
Samsung/tizen-docs: docs/application/dotnet/guides/nui/nui-components/Tab.md

![Tab](./media/tab.png) ![Tab](./media/tab2.png)

> [!NOTE]

> > Tab is deprecated in API8 (Tizen 6.0) and will be removed in API10.

Reviewer: Please rephrase to:

Tab is deprecated since Tizen 6.0 and will be removed after two releases.

> ## Add namespace using
> Tab is defined in namespace Tizen.NUI.Components.

Reviewer: Please verify the following suggestion for technical accuracy.

```suggestion
## Add namespace
To implement tab, add namespace `using Tizen.NUI.Components` in your application:
```
